customizable.html

```diff
@@ -126,7 +126,23 @@ <h2 id="resources">Learn more</h2>
             <li><a href="https://www.w3.org/WAI/intro/people-use-web/stories.html#classroomstudent">Ms. Olsen, Classroom student with attention deficit hyperactivity disorder (ADHD) and dyslexia</a></li>
           </ul>
         </li>
-        <li><strong>WCAG 2.0:</strong> <a href="https://www.w3.org/WAI/WCAG20/quickref/?tags=text">Success Criteria relating to "text"</a></li>
+        <li><strong>User Needs:</strong>
+          <ul>
+            <li><a href="http://www.w3.org/TR/low-vision-needs/"> Accessibility Requirements for People with Low Vision            </a></li>
```

Comment:
I think it is better to move this down after WCAG and UAAG for a variety of reasons.